e2e/test/scenarios/visualizations-tabular/smartscalar-trend.cy.spec.js

```diff
@@ -364,7 +364,7 @@ describe("scenarios > visualizations > trend chart (SmartScalar)", () => {
     cy.icon("arrow_down").should(
       "have.css",
       "color",
-      Color(colors.error).string(),
+      Color(colors.error).rgb().string(),
```

Comment:
The actual value produced in the CSS is in `rgb` format. I guess the reason this changes is because error colors are now defined in `hlsa`, so the `.string()` returns `hsl` which doesn't match with the expected format.